samples/wcfapp/README.md

```diff
@@ -0,0 +1,59 @@
+# WCF Docker Container Samples
+These samples demonstrate how to dockerize WCF services, either IIS-hosted or self-hosted. A simple "hello world" service contract is used in all samples for both HTTP and NET.TCP transport bindings.
+
+## Prepare Your Environment
```

Comment:
I think the best UX would be for the format of this readme to mirror the other samples where it make sense.  As an example, there should be a section that mirrors https://github.com/Microsoft/dotnet-framework-docker/blob/master/samples/dotnetapp/README.md#getting-the-sample.